src/test/codegen/thread-local.rs

```diff
@@ -0,0 +1,50 @@
+// compile-flags: -O
+// aux-build:thread_local_aux.rs
+// ignore-windows FIXME(#84933)
```

Comment:
This should have been `ignore-msvc` because now we enable it for `windows-gnu` without testing.